src/libraries/icubmod/embObjMotionControl/eomcParser.h

```diff
@@ -370,6 +370,7 @@ class Parser
     bool getCorrectPidForEachJoint(PidInfo *ppids/*, PidInfo *vpids*/, TrqPidInfo *tpids);
     bool parsePidUnitsType(yarp::os::Bottle &bPid, yarp::dev::PidFeedbackUnitsEnum  &fbk_pidunits, yarp::dev::PidOutputUnitsEnum& out_pidunits);
 
+    bool checkJointTypes(PidInfo *pids, std::string pid_type);
```

Comment:
This ought to be `const std::string& pid_type`.

Reply:
done!

Reply:
It's important the `&` as it needs to be a reference.

Reply:
oops, fixed! (this time for real)